pandas/core/series.py

```diff
@@ -781,9 +781,20 @@ def reshape(self, *args, **kwargs):
 
         return self.values.reshape(shape, **kwargs)
 
-    iget_value = _ixs
-    iget = _ixs
-    irow = _ixs
+    def iget_value(self, i, axis=0):
```

Comment:
Can you add here a docstring like:
```
"""DEPRECATED. Use ``.iloc[i]`` or ``.iat[i]`` instead"""
```

to every of those functions? (and maybe also for the ones in frame.py) 
Because they will still appear on the automatically generated API pages, and then it is more clear you shouldn't use instead of no docstring.